tests/helpers.test.js

```diff
@@ -0,0 +1,82 @@
+import { afterEach, describe, expect, it, jest } from '@jest/globals';
+import merge from 'lodash.merge';
+import * as github from '@actions/github';
+import * as helpers from '../src/helpers';
+
+jest.mock('@actions/core');
+jest.mock('@actions/github'); // NOTE(dabrady) Manually mocked at: `__mocks__/@actions/github.js`
+
+describe('#requestReviewers', () => {
+  afterEach(jest.clearAllMocks);
+
+  var payload = {
```

Comment:
Clarification, I haven't excluded test files from linting, I'm excluded them from being linted by a particular plugin which focuses on an opinionated use of arrow functions.

To your point about `const` vs `var`, `const` won't have the effect you think it will here. Objects are mutable: it doesn't matter what box you put them in. `const` simply prevents _reassignment_, not _mutation_. For that reason, I avoid using `const` in general, and never use `const` for mutable data types like objects and arrays unless I'm defining a 'shared constant', in which case I also use [`Object.freeze()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) and make the name `ALL_CAPS`.

(This is actually one of the few things I've ever blogged about 😆 If you're interested in reading, I have a [3-part series on DEV.to](https://dev.to/daniel13rady/declaring-variables-in-javascript-31ch).)

Reply:
I just read the whole vars part and will give the other ones a go. Might have some questions afterwards 😅 there have been many a heated discussions on the use of var/let and const.